qa/TL1_custom_cpp_lib/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+do_once() {
+  apt-get update
+  apt-get -y install cmake
+}
+
+test_body() {
+    export TMP_PATH="$(mktemp -d)"
+    pushd $TMP_PATH
+    cmake -D CUDA_TARGET_ARCHS=60 .
```

Comment:
```suggestion
    cmake .
```
and maybe better to have a build directory
```
cd build
cmake ..
```

Reply:
I went without build directory, cause I guess it doesn't really matter, it works automatically anyway